daffodil-tdml-processor/src/test/scala/org/apache/daffodil/tdml/TestTDMLRunner.scala

```diff
@@ -68,34 +69,30 @@ class TestTDMLRunner {
                       </parserTestCase>
                     </testSuite>
 
-    using(new java.io.FileWriter(tmp)) {
-      fileWriter =>
-        fileWriter.write(testSchema.toString())
-    }
-
-    // compileAllTopLevel must be true to enable caching
-    lazy val ts = new DFDLTestSuite(testSuite, true, true, true)
+    lazy val runner = new Runner(testSuite)
 
     // Run the non-validation test first, this will compile, serialize, and
     // cache the schema
-    ts.runOneTest("test3a")
+    runner.runOneTest("test3a")
 
     // Run the validation test second. This should recompile and cache the
     // schema separately from from the first test. If it ended up using the
     // non-validation cached Processor, an exception would be thrown since it
     // was serialized and that does not work with validation="on"
-    ts.runOneTest("test3b")
-  }
+    runner.runOneTest("test3b")
 
-  val testSchema = SchemaUtils.dfdlTestSchema(
-    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>,
-    <dfdl:format ref="tns:GeneralFormat"/>,
-    <xs:element name="data" type="xs:int" dfdl:lengthKind="explicit" dfdl:length="{ xs:unsignedInt(2) }"/>)
+    runner.reset
+  }
 
   @Test def testTDMLParseSuccess(): Unit = {
 
-    val testSuite = <ts:testSuite xmlns:ts={ tdml } suiteName="theSuiteName">
-                      <ts:parserTestCase ID="some identifier" name="testTDMLParseSuccess" root="data">
+    val testSuite = <ts:testSuite xmlns:ts={ tdml } suiteName="theSuiteName" xmlns:xs={ xsd } xmlns:dfdl={ dfdl } xmlns:tns={ example }>
+                      <ts:defineSchema name="mySchema">
```

Comment:
This is a nice cleanup, as all these tests pre-dated our introduction of the ts:defineSchema element to tdml.